basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,232 @@
 const helpers = require('./project-1');
 
 // start testing!
+describe('project-1', () => {
+    describe('multiplyByTen', () => {
+      it('returns NaN when given a non numeric value', () => {
+        expect(helpers.multiplyByTen(undefined)).toBeNaN();
+        expect(helpers.multiplyByTen('two')).toBeNaN();
+      });
+  
+      it('should return the given number multiplied by 10', () => {
+          expect(helpers.multiplyByTen(4)).toBe(40);
+          expect(helpers.multiplyByTen(0)).toBe(0);
+          expect(helpers.multiplyByTen(-1)).toBe(-10);
+      });
+    });
+
+    describe('subtractFive', () => {
+        it('returns NaN when given a non numeric value', () => {
+            expect(helpers.subtractFive(undefined)).toBeNaN();
+            expect(helpers.subtractFive('two')).toBeNaN();
+          });
+        it('should subtract by five', () => {
+            expect(helpers.subtractFive(0)).toBe(-5);
+            expect(helpers.subtractFive(7)).toBe(2);
+            expect(helpers.subtractFive(-2)).toBe(-7);
+        });
+    });
+
+    describe('areSameLength', () => {
```

Comment:
This is great, initializing test cases. :+1: What do you think about adding cases for bad data?